Backend/Backend/Controllers/Responses/PodcastResponses.cs

```diff
@@ -45,11 +45,13 @@ public PodcastResponse(Podcast p, string domainUrl)
         CoverArtUrl = domainUrl + string.Format("podcast/{0}/getCoverArt", p.Id);
         Tags = p.Tags;
         IsExplicit = p.IsExplicit;
-        AverageRating = p.AverageRating;
-        TotalRatings = p.TotalRatings;
         Type = p.GetPodcastTypeString();
         PodcasterId = p.PodcasterId;
         Episodes = p.Episodes.Select(e => new EpisodeResponse(e, domainUrl)).ToList();
+        Ratings = p.Ratings.Select(r => new RatingResponse(r)).ToList();
+        TotalRatings = (ulong) Ratings.Where(r=>r.Rating!=0).Count();
+        if(TotalRatings>0)
```

Comment:
is this initialised when total ratings are 0? in that case they should be 0

Reply:
If total ratings are 0, then an empty list is returned.